### ISSUE TITLE
MiT-B3 is much better than MiT-B4

### ISSUE THREAD

**Issue by 5as4as** [lhoyer/DAFormer#47]
Dear authors, thank you for your outstanding work. I have encountered results in reproducing your work that make me puzzled: in the GTAV->Cityscapes experiment, when the backbone network is MiT-B5, I get results similar to the paper (68.3); when the backbone network is MiT-B4, I get an mIoU of **66.69**; when the backbone network is MiT-B3, I get an mIoU of **67.91**. I am confused why MiT-B3 is so much better than MiT-B4. Have you conducted similar experiments? What are the results like?

**Comment by lhoyer** [comment]
Dear @5as4as,

Thank you for your interest in our work. We conducted some experiments with smaller MiT backbones with the baseline UDA (without RCS and FD) as shown in Tab. 3. However, we did not try smaller MiT backbones with RCS, FD, and the proposed decoder. I agree that your results are somewhat unexpected. It shows that RCS, FD, and the proposed decoder give an even greater improvement for the smaller backbone when compared to Tab. 3. However, I don't know why the results for MiT-B3 are better than MiT-B4. Have you run the experiment on multiple seeds to exclude random fluctuations in the performance?

Best,
Lukas
